cypress/e2e/createProject.cy.ts

```diff
@@ -25,7 +25,7 @@ describe('Create Project', () => {
     cy.findByTestId(createProjectFormIds.notification.success).should('exist');
   });
 
-  it('[NOK]: Admin wants to create a project with short name', () => {
+  it.skip('[NOK]: Admin wants to create a project with short name', () => {
```

Comment:
why is this skipped? 🤔

Reply:
I haven't understood this test
and we mark it as `NOK`

Reply:
the NOK means that the test is for a situation in the app that should not work for 
in other words, the admin wants to create a project with a short name but that's not ok (NOK)

Reply:
but the test is ok, right?
I mean we're typing 3 letters word

Reply:
yes if I understood correclty 
we're typing 3 letters which should result in a error (NOK)

Reply:
but 3 letters word is ok
I think Alireza just changed the title